README.md

```diff
@@ -1,21 +1,18 @@
 # post-covid-respiratory
```

Comment:
All the removed content from this file needs to be put back in.

Reply:
Done